src/renderer/services/eth2/client/prysm/prysm.ts

```diff
@@ -19,14 +23,35 @@ export class PrysmEth2ApiClient extends AbstractApiClient implements IValidatorB
 
     public constructor(options: IBeaconClientOptions) {
         super();
+        options.urlPrefix = `${options.urlPrefix}${apiPrefix}`;
         this.url = options.urlPrefix;
         this.config = options.config;
         this.beacon = new PrysmBeaconApiClient(options);
         this.validator = new PrysmValidatorApiClient(options);
     }
-    
+
+    public static getPrysmBeaconClient(url: string, network: string): PrysmEth2ApiClient | null {
+        const networkConfig = getNetworkConfig(network);
+        if (!networkConfig) {
+            return null;
+        }
+
+        return new PrysmEth2ApiClient({
+            urlPrefix: url,
+            config: networkConfig.eth2Config
+        });
+    }
+
     public async getVersion(): Promise<string> {
         return (await this.beacon.getClientVersion()).toString("ascii");
     }
 
+    public async isSyncing(): Promise<boolean> {
```

Comment:
Do you really need that util method

Reply:
This one seems to have 2 types of responses so yes :P

Reply:
Yeah but you can handle responses in container like
```javascript
const status = await api.beacon.getSyncingStatus()
if(!status) {
"synced"
} else {
"syncing"
}
```